internal/service/autoscaling/policy.go

```diff
@@ -111,9 +381,10 @@ func ResourcePolicy() *schema.Resource {
 										},
 									},
 									"predefined_scaling_metric_specification": {
-										Type:     schema.TypeList,
-										Optional: true,
-										MaxItems: 1,
+										Type:          schema.TypeList,
+										Optional:      true,
+										MaxItems:      1,
+										ConflictsWith: []string{"predictive_scaling_configuration.0.metric_specification.0.customized_scaling_metric_specification"},
 										Elem: &schema.Resource{
 											Schema: map[string]*schema.Schema{
 												"predefined_metric_type": {
```

Comment:
On line 393 where the strings are hardcoded in the ValidateFunc, we prefer to dive into the SDK and use the list of values defined in the SDK.

ValidateFunc: validation.StringInSlice(autoscaling.PredefinedScalingMetricType_Values(), false),